src/lib/Components/Inbox/Bids/ActiveBid.tsx

```diff
@@ -95,6 +99,10 @@ class ActiveBid extends React.Component<RelayProps, State> {
     }
   }
 
+  handleTap() {
+    SwitchBoard.presentNavigationViewController(this, this.props.bid.active_bid.sale_artwork.artwork.href)
```

Comment:
I recall talking about routing to an auction lot VC for the bid in question but we can update that once we have the URI